SillyBusinessInc/Moldbreaker
# Feat/#228 yop UI

## Purpose of this PR:
when killed all enemies go to the yeast of power cube and walk into it, it will display a popup and click on confirm

## How to test:
go to the Menu scene in the global folder. kill all enemies (or delete the melee enemy gameobjects in the hierarchy) and the yeast of power cube will appear. walk to it and click on confirm

if you want to change the upgrade. you need to go to level scene and go to YeastOfPower > Interactable Object and change the Interactable Upgrade UI Option field to whatever powerup you want it to be

### links: 
[Ticket](https://github.com/SillyBusinessInc/Moldbreaker/issues/228)
